README.md

```diff
@@ -18,38 +18,40 @@ Usage by example
 ----------------
 
 ```Erlang
-1> {ok, Pid} = ered:start_link([{"localhost", 6379}], []).
+1> {ok, _} = application:ensure_all_started(ered, temporary),
+2> {ok, Pid} = ered:connect_cluster([{"localhost", 6379}], []).
 {ok,<0.164.0>}
-2> ered:command(Pid, [<<"SET">>, <<"mykey">>, <<"42">>], <<"mykey">>, 5000).
+3> ered:command(Pid, [<<"SET">>, <<"mykey">>, <<"42">>], <<"mykey">>, 5000).
 {ok,<<"OK">>}
-3> ered:command_async(Pid, [<<"GET">>, <<"mykey">>], <<"mykey">>, fun(Reply) -> io:format("Reply: ~p~n", [Reply]) end).
+4> ered:command_async(Pid, [<<"GET">>, <<"mykey">>], <<"mykey">>, fun(Reply) -> io:format("Reply: ~p~n", [Reply]) end).
 ok
 Reply: {ok,<<"42">>}
-4> ered:stop(Pid).
+5> ered:stop(Pid).
```

Comment:
Hum, just starting to think if this should this be `ered:disconnect/1`?  Like start/stop, connect/disconnect and open/close.. 

I believe the example at least should be changed to `ered:close` to match the current code.

Reply:
gen_tcp and ssl use connect/close, so I think it's good to follow that convention.

```suggestion
5> ered:close(Pid).
```